File: go/wam/4970.go
const WamSupportAiSession = 4970 // channel: regular

type SupportAiSessionEvent struct {
	SupportAiEventType SUPPORTAIEVENTTYPE `wam:"support_ai_event_type"` // field 1
	SupportAiErrorCode int64 `wam:"support_ai_error_code"` // field 3
	SupportAiErrorMessage string `wam:"support_ai_error_message"` // field 4
	CitationCmsId string `wam:"citation_cms_id"` // field 5
}

File: ts/wam/4970.ts
export const WamSupportAiSession = 4970 as const

export interface SupportAiSessionEvent {
  /** field 1, wire `support_ai_event_type` */
  supportAiEventType?: SUPPORTAIEVENTTYPE
  /** field 3, wire `support_ai_error_code` */
  supportAiErrorCode?: number
  /** field 4, wire `support_ai_error_message` */
  supportAiErrorMessage?: string
  /** field 5, wire `citation_cms_id` */
  citationCmsId?: string
}